react-backend/routes/cb_name_check.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router();
+const cbDetails = require('../database/queries/cb/cb_details');
+
+router.get('/', (req, res, next) => {
+  cbDetails(req.app.get('client:psql'), req.auth.cb_id)
```

Comment:
Here you don't need to do a db query as both of the pieces of info you're after are added onto `req.auth` using middleware:

https://github.com/TwinePlatform/DataPower/blob/9ac8d1934aced9d2412c9b9943a4ea89d0f613ab/react-backend/routes/cbauthentication/mw_is_authenticated.js#L14-L23

Reply:
Sweet, will do